### ISSUE TITLE
FetchOHLCV Return Both QuoteVolume and BaseVolume

### ISSUE THREAD

**Issue by maoxiaoyun1963** [ccxt/ccxt#20637]
### Preliminary Checks

- [X] I have already searched for [existing issues](https://github.com/ccxt/ccxt/issues) and confirmed that this issue is not a duplicate

### Is your feature request related to a problem? Please describe

In ohlcv(kline) trading, a very common operation is to calculate each kline's VWAP as an estimation of average deal price. However, ccxt only return either baseVolume or quoteVolume alone, making it impossible to calculate VWAP. Please add both baseVolume and quoteVolume to OHLCV data. Thanks.

### Describe the solution you'd like

Most exchange return both information to the users. CCXT could pass along this information. For those exchange that only return one of required information, CCXT could return nan or None to signal the users such information is not provided by exchagne. 

### Describe alternatives you've considered

CCXT could also return VWAP directly to users.

**Comment by sc0Vu** [comment]
Hi @maoxiaoyun1963

There is vwap in ticker structure, the value wouldn't be empty if exchange return. @carlosmiei how do you think we add another volume in ohlcv data?

**Comment by maoxiaoyun1963** [comment]
True, it would be neat if we can obtain this information in ohlcv data instead of needing to combine multiple data sources.

Also, if it is possible, please consider adding buyer init quote volume and buyer init seller volume as well. It is extremely useful in trading. (https://binance-docs.github.io/apidocs/spot/en/#kline-candlestick-data)

Thanks.

**Comment by maoxiaoyun1963** [comment]
Also, VWAP in Ticker is in 24 hour interval. We might need VWAP in other intervals as well.

**Comment by rhythttry** [comment]
fdj

**Comment by leaf-hust** [comment]
Hi @sc0Vu @carlosmiei 
Can the FetchOHLCV method add the above fields to facilitate our use?

**Comment by WillPCowan** [comment]
Quote volume in OHLCV (when exchanges provide it) would be extremely useful -- or if this is tricky then providing "info" property with exchange response so we can get it ourselves would be really nice.
